src/PKSim.Core/Snapshots/Mappers/CompoundMapper.cs

```diff
@@ -91,9 +91,17 @@ private async Task updateAlternatives(ModelCompound compound, Alternative[] snap
          //Remove all alternatives except calculated ones
          alternativeGroup.AllAlternatives.ToList().Where(x => !x.IsCalculated).Each(alternativeGroup.RemoveAlternative);
 
+         //Reset the default flag that will be read from snapshot
+         alternativeGroup.AllAlternatives.Each(x=>x.IsDefault = false);
```

Comment:
Before loading alternatives, reset all isdefault to false (calcualted was is set to is defualt true, that's why we had two is defualt)